resources/assets/lib/notification-widget/main.tsx

```diff
@@ -187,7 +186,15 @@ export default class Main extends React.Component<Props, State> {
     });
 
     if (nodes.length === 0) {
-      const transKey = this.controller.currentFilter == null ? 'notifications.all_read' : 'notifications.none';
+      let transKey = 'notifications.loading';
+      if (!core.socketWorker.hasConnectedOnce) {
+        transKey = 'notifications.loading';
+      } else if (this.controller.currentFilter == null) {
+        transKey = 'notifications.all_read';
```

Comment:
similar to the count display, this will be shown when websocket is connected but the initial data is still being downloaded

Reply:
this one is still not fixed :sleeping: